.buildkite/dagster-buildkite/dagster_buildkite/package_spec.py

```diff
@@ -291,3 +288,22 @@ def requirements(self):
         ]
         install = self.distribution.install_requires
         return extras + install
+
+    @property
+    def skip_reason(self):
```

Comment:
nit: this naming feels a little off given what actually happening, `should_skip`, `should_skip_with_reason`, `reason_if_should_skip` 
those names  aren't amazing either, maybe youll come up with something better. feedback applies equally to the arg on build_tox_step

Reply:
I'm actually going to capture a lot of these renames in another PR to to save myself from needing to rebase a bunch of in flight work.